### PR TITLE
[13] Add a link to the cart to the header

### PR DESCRIPTION
**Pull request by iggysav**
Added a link to the cart, cart icon, dynamic display of the number of orders in the cart.

### CODE REVIEW

File: app/assets/stylesheets/components/cart/_cart_link.scss [hunk 1]
```diff
@@ -11,14 +11,16 @@
   background-position: bottom;
   position: relative;
 
+  //https://trello.com/c/CQKgoAUG/13-add-a-link-to-the-cart-to-the-header
```
Comment (fooooxmr) [inline app/assets/stylesheets/components/cart/_cart_link.scss:14]: Why did you change existing CSS? Are you sure that this will not affect other places?

File: app/views/layouts/_top_bar.html.erb [hunk 2]
```diff
@@ -13,7 +13,16 @@
     <%= link_to I18n.t('spree.login'), login_path, class: 'auth-link' %>
   <% end %>
 
+<!--  https://trello.com/c/CQKgoAUG/13-add-a-link-to-the-cart-to-the-header-->
   <div class="cart-link" id="link-to-cart" data-top-bar-target="cartLink">
+    <%= link_to cart_path do %>
+      <div class="cart-info" id="cart-info"> <!-- заменили класс badg на cart-info -->
```
Comment (fooooxmr) [inline app/views/layouts/_top_bar.html.erb]: Please, do not use Russian comments.

**Review by fooooxmr — APPROVED** [review]

File: app/assets/stylesheets/components/cart/_cart_link.scss (hunk 1 above)
Comment (iggysav) [reply app/assets/stylesheets/components/cart/_cart_link.scss:14]: It's okay. This css only applies to the linked cart. I only changed the backgroud color from blue to red.

File: app/views/layouts/_top_bar.html.erb (hunk 2 above)
Comment (iggysav) [reply app/views/layouts/_top_bar.html.erb]: I apologize. It was for myself, as there were a lot of changes. To make it easier to rollback. I forgot to remove this comment.

**Review by fooooxmr — APPROVED** [review]

**Review by olegbrest1 — APPROVED** [review]
